extension/Config.jsm

```diff
@@ -0,0 +1,69 @@
+/* to use:
+
+- Recall this file has chrome privileges
+- Cu.import in this file will work for any 'general firefox things' (Services,etc)
+  but NOT for addon-specific libs
```

Comment:
No, it is possible to use `XPCOMUtils.defineLazyModuleGetter()` to import `resource://` files as specified by the `chrome.manifest`. Updated comments to reflect this.